apcd-cms/src/apps/utils/registrations_data_formatting.py

```diff
@@ -1,22 +1,23 @@
-def _set_registration(reg, reg_ents, reg_conts):
-    org_types = {
-            'carrier': 'Insurance Carrier',
-            'tpa_aso': 'Plan Administrator¹ (TPA/ASO)',
-            'pbm': 'Pharmacy Benefit Manager (PBM)'
+def _get_orgtypes():
+    return {
+        'carrier': 'Insurance Carrier',
+        'tpa_aso': 'Plan Administrator¹ (TPA/ASO)',
+        'pbm': 'Pharmacy Benefit Manager (PBM)',
     }
+
+
+def _set_registration_for_listing(reg):
+    org_types = _get_orgtypes()
     return {
-            'biz_name': reg[5],
-            'type': org_types[reg[4]] if (reg[4] and reg[4] in org_types.keys()) else None,
-            'location': '{city}, {state}'.format
-                (
-                    city=reg[7],
-                    state=reg[8]
-                ),
-            'reg_status': reg[3].title() if reg[3] else None,
-            'reg_id': reg[0],
-            'year': reg[10],
-            'view_modal_content': _set_modal_content(reg, reg_ents, reg_conts, org_types)
```

Comment:
removing this - creating un-necesary loading of data. With react, we can get the data we want on-demand